include/yaml-cpp/node/convert.h

```diff
@@ -275,6 +468,27 @@ struct convert<std::array<T, N>> {
   }
 };
 
+// std::bitset
+template <std::size_t N>
+struct convert<std::bitset<N>>
+{
+  using value_type = std::bitset<N>;
+  
+  static Node encode(const value_type& rhs)
+  {
+    return convert<std::string>::encode(rhs.to_string());
+  }
+
+  static bool decode(const Node& node, value_type& rhs)
+  {
+    std::string representation;
+    if (!convert<std::string>::decode(node, representation)) return false;
+    if (representation.size() != N) return false;
+    rhs = value_type(representation);
```

Comment:
"If any characters examined in str are not zero or one, it throws std::invalid_argument."

... but decode is supposed to return false.